tools/build-debian-package.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
+#
+# Written by Daniil Baturin, 2018
+# This file is public domain
+
+echo "Cleaning up the build"
+git clean -dfx
+git reset --hard
```

Comment:
Isn't it a dangerous part to clean/reset any changes before building the package?
I would assume someone who builds their own package has most likely modified the source, so
just resetting everything which is not committed might be dangerous.
(One possible alternative would be to give a warning if the git is not clean)

Reply:
I agree, it was over the top. I've fixed it in the updated PR.